### DISCUSSION TITLE
body background can't change

### DISCUSSION THREAD

**Discussion by madfortech** [saadeghi/daisyui#1552] [General]
![Capture](https://user-images.githubusercontent.com/21261716/214236044-90fed44f-3af4-46f8-9dfe-3599e0cec74d.PNG)
![Capture-1](https://user-images.githubusercontent.com/21261716/214236166-fd94a2dc-964e-48da-bafe-524ce86a0a85.PNG)

<body class="bg-red-900">
        @include('layouts.nav')
        <main>
            @yield('content')
        </main>
</body>
</html>
how i remove default bg color actually install dailyui open my browser background color dark grey i want change  ? any one help me

**Comment by saadeghi** [comment]
daisyUI applies the [`base-100` color](https://daisyui.com/docs/colors/) to the background of `<html>` tag (which is a dark color with the `dark` theme)  
You can simply set your custom colors to make a new theme (see [our theme generator](https://daisyui.com/theme-generator/)) or if you want to set a new color, you can add the class name to the `<html>` tag. Adding the class names to the `<body>` like you did works too but it would be a `<body>` (with your custom color) inside the `<html>` of course.

Let me know if you have any questions.
